### PR TITLE
add: main greating2

### PR DESCRIPTION
**Pull request by Dsypasit**

### CODE REVIEW

**Comment by golfzakrub** [comment]
Dee kub

**Comment by golfzakrub** [comment]
Pai tum ma mai

File: main.py [hunk 1]
```diff
@@ -1,2 +1,3 @@
 import hello
-hello.greeting()
```
Comment (golfzakrub) [inline main.py:2]: Why you doing this?

**Review by Dsypasit — COMMENTED** [review]
sadlfkjaskdlfj

**Pull request closed without merging**